src/middleware/jira-admin-permission-middleware.ts

```diff
@@ -0,0 +1,46 @@
+import { NextFunction, Request, Response } from "express";
+import { Installation } from "models/installation";
+import { JiraClient } from "models/jira-client";
+import { booleanFlag, BooleanFlags } from "config/feature-flags";
+
+export const setJiraAdminPrivileges = async (req: Request, claims: Record<any, any>, installation: Installation): Promise<void> => {
```

Comment:
nit the name sounds like we are rising up the flag.  Maybe something like "fetchAndSaveUserAdminStatus" ? I know it is not ideal too...